docs/operations/etcd/cluster-migration-node-to-pod.md

```diff
@@ -0,0 +1,344 @@
+---
+title: "Etcd cluster migration"
+---
+
+
+This procedure explains how to migrate data from an etcd cluster running
+outside a Kubernetes cluster towards an installation using the Ondat etcd
+Operator. As a result the Ondat cluster will use the etcd running as Pods in
+Kubernetes.
+
+## Prerequisites
+
+- Kubectl
+- Helm
+
+It is assumed that both etcd clusters in this procedure are using mTLS.
+
+## Procedure
+
+### Option A - Manual process
+
+1. Backup the TLS artifacts from the current etcd cluster (if current etcd uses mTLS)
+    The Secret with the TLS material is usually named `storageos-etcd-secret`
+    or `etcd-client-tls`
+
+    ```bash
+    $ kubectl get secret \
+        -n storageos \
+        -o yaml \
+        etcd-client-tls > etcd-storageos-tls-secret-backup.yaml
+    ```
+
+1. Deploy etcd cluster in Kubernetes
+
+    ⚠️  It is required to select a storageClass other than ondat/storageos to
+    run etcd in the cluster. If there is none available in the cluster, you can
+    run the following to deploy a [Local Path
+    Provisioner](https://github.com/rancher/local-path-provisioner) to provide
+    local storage for Ondat's embedded `etcd` cluster operator deployment.
+
+    __(Optional) Deploy Local path storageClass__
+
+    ```bash
+    kubectl apply --filename="https://raw.githubusercontent.com/rancher/local-path-provisioner/v0.0.21/deploy/local-path-storage.yaml"
+    ```
+
+    __Deploy etcd__
+
+    ```bash
+    # Add ondat charts repo.
+    $ helm repo add ondat https://ondat.github.io/charts
+    # Install the chart in a namespace.
+    $ helm install storageos-etcd ondat/etcd-cluster-operator \
+        --namespace etcd-operator \
+        --create-namespace \
+        --set ondat.secret=storageos-etcd-secret-incluster \
+        --set cluster.storageclass=standard # Choose the one according to your cluster
+    ```
+
+    > ⚠️  The Secret name for the etcd client certificates is amended to avoid
+    > using the same Secret name as the one used for the old etcd cluster
+
+    > If the chart is uninstalled the Secret in the storageos namespace with
```

Comment:
I don't believe this is the case anymore, since this PR:
https://github.com/storageos/etcd-cluster-operator/pull/46